lib/ruby_lsp/requests/on_type_formatting.rb

```diff
@@ -60,7 +60,7 @@ def perform
         when "|"
           handle_pipe if @document.syntax_error?
         when "\n"
-          if (comment_match = @previous_line.match(/^#(\s*)/))
+          if (comment_match = @previous_line.match(/^#(?!:)(\s*)/))
```

Comment:
Let's add a comment here about the RBS syntax?

Reply:
Added 👍